engines/scumm/string.cpp

```diff
@@ -454,6 +461,88 @@ bool ScummEngine::newLine() {
 	return true;
 }
 
+void ScummEngine::fakeBidiString(char *ltext, bool ignoreVerb) {
+	int ll = 0;
```

Comment:
It would be useful if you describe the purpose of this method, and perhaps even what is going on, in the comments.